utils/src/main/kotlin/OkHttpClientHelper.kt

```diff
@@ -92,7 +92,8 @@ object OkHttpClientHelper {
      * Apply HTTP proxy settings from the "https_proxy" or "http_proxy" environment variables.
      */
     fun OkHttpClient.Builder.applyProxySettingsFromEnv(): OkHttpClient.Builder {
-        val proxyUrl = Os.env["https_proxy"]?.addMissingHttpProtocol("https")
+        // Note that even HTTPS proxies use "http://" as the protocol!
+        val proxyUrl = Os.env["https_proxy"]?.addMissingHttpProtocol("http")
```

Comment:
It seems that from here on `addMissingHttpProtocol` is always called with "http"  as parameter, so it could be hardcoded. This would also better fit the name `addMissingHttpProtocol`.